modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15305,7 +15305,7 @@ service=Service
 service-access-policy-entry-default-asset-entry-title=Public Access to the Asset Entry API
 service-access-policy-entry-default-asset-tag-title=Public Access to the Asset Tag API
 service-access-policy-entry-default-calendar-title=Public Access to the Calendar Search API
-service-access-policy-entry-default-captcha-title=Public Access to Captcha API
+service-access-policy-entry-default-captcha-title=Public Access to CAPTCHA API
```

Comment:
why is change in language.properties in buildlang commit ?

Reply:
Fixed it. Thanks